docs/apis/streaming/fault_tolerance.md

```diff
@@ -176,6 +176,11 @@ state updates) of Flink coupled with bundled sinks:
         <td></td>
     </tr>
     <tr>
+        <td>Cassandra sink</td>
+        <td>exactly once</td>
```

Comment:
I mean an at most once system will give you also exactly once processing guarantees under the assumption that nothing fails.